pkg/remote/remote.go

```diff
@@ -134,6 +127,11 @@ func WithCleanup() heartbeat.HandleOption {
 	}
 }
 
+func removeWithLogging(file string) {
```

Comment:
Let's rename it to `deleteLocalFile`? It has a bug where error is not being validated and always being logged.